### DISCUSSION TITLE
Optional `target` parameter for Portal within @mantine/notifications

### DISCUSSION THREAD

**Discussion by bentron2000** [mantinedev/mantine#2366] [Feature requests]
I have a desire for the Portal used by `NotificationsProvider` to be different to the one used by `Overlay` (or other mantine components using portal under the hood)

I propose an optional `target` prop for the `NotificationsProvider` to be forwarded to its inner Portal component to achieve this result.

I have tried nested `MantineProvider` components with `inherit` and just the following in the theme but with no luck:
``` typescript
{
   components: {
      Portal: {
         defaultProps: {
            target: ref.current // a ref to where I want the notifications to appear
         }
      }
   }
}
```

If there is another way to achieve this other than the proposal above - let me know. 

:)

**Comment by rtivital** [comment]
I'll add `target` prop to NotificationsProvider in one of the next releases.
